The hardware requirements for MDE are confusing (#11346)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/minimum-requirements.md

Devices on your network must be running one of these editions.

> The hardware requirements for Defender for Endpoint on devices are the same as the one of the Windows Operating System it self (meaning they are not in addition of requirements of the Operating System) for the supported editions.

Reviewer: ```suggestion
The hardware requirements for Defender for Endpoint on devices are the same as the requirements for the Windows operating system itself (that is, they are not in addition to the requirements for the operating system) for supported editions.
```
